client/testdata/accounts_wrong_format_fixture.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<accounts>
+    <some_attr>
```

Comment:
Might consider moving these to the fake package and exposing if we foresee the need for them to be accessed somewhere else (the operator ) for testing purposes?

Reply:
Following golang testing guidelines. Tests fixtures can go in `testdata` folder which `go build` ignores

Reply:
Sure. My point was just that for unit testing in the caller, it might be useful to have access to some mock responses. Of course if that doesn't make sense here, then this is the preferred solution.